airflow/config_templates/default_celery.py

```diff
@@ -106,11 +107,11 @@ def _broker_supports_visibility_timeout(url):
     )
 except Exception as e:
     raise AirflowException(
-        f"Exception: There was an unknown Celery SSL Error. Please ensure you want to use SSL and/or have "
-        f"all necessary certs and key ({e})."
+        "Exception: There was an unknown Celery SSL Error. Please ensure you "
+        "want to use SSL and/or have all necessary certs and key ({}).".format(e)
```

Comment:
Same question. Looks like you’re trying to avert f-strings and I don’t get the motivation.